patchwork-entity/build.gradle

```diff
@@ -0,0 +1,10 @@
+archivesBaseName = "patchwork-entity"
+version = getSubprojectVersion(project, "0.1.0")
+
+dependencies {
+	implementation project(path: ':patchwork-api-base', configuration: 'dev')
+}
+
+loom {
+	accessWidener "src/main/resources/patchwork-MobEntity.accesswidener"
```

Comment:
Rename the AW to match the module name

Reply:
Oh it was my understanding that the file name should match the class that you are widening.
As for the module name, I asked on discord and patchwork-entity seemed acceptable. There was mention of patchwork-extensions-entity. I'm good with either just let me know.